### ISSUE TITLE
Inefficient android project structure

### ISSUE THREAD

**Issue by mohammad-haque** [terrakok/Compose-Multiplatform-Wizard#43]
This wizard produce this when android and ios is the target:
`composeApp`
    - `androidMain`
    - `iosMain`
    - `commonMain`
`iosApp`

Notice no `androidApp` outside the `composeApp` and this not ideal because a dedicated `androidApp` project folder is useful for many reasons, including, having separate debug resource for debug app, alternative implementation for debug app (e.g. AndroidManifest file) etc.

Is there any plan to bring back `androidApp` folder?

**Comment by terrakok** [comment]
No. I don't have such plans. I think a single module for whole compose app is enough. It is not a shared library, it is the final app.
